ui/raidboss/data/triggers/byakko-ex.js

```diff
@@ -6,36 +6,55 @@
     {
       id: 'ByaEx Heavenly Strike',
       regex: / 14:27DA:Byakko starts using Heavenly Strike on (\y{Name})/,
+      regexDe: / 14:27DA:Byakko starts using Himmelszorn on (\y{Name})/,
```

Comment:
Did you mean to include all the byakko stuff in this patch too or is this intended to just be one megapatch and I should close the other one?

Reply:
I'm not sure what you mean.  You said different things in different patches.  It looks like you fixed things here, so maybe we should close the other patch and just have this patch?

Reply:
Hmmmm... maybe that might work. I'm not very sure, because:
- with the byakko-translations I did some changes on my local byakko.js and eureka.js only. I uploaded these as byakko-translations branch afterwards
- with the german-completion-patch I did changes afterwards on top of that byakko-translations branch
So if github compares all lines and replaces the difference you can simply close the other pull request and ignore it. If it only changes what was changed within this branch and it's edit you'd have to merge the other one first though.